cumulusci.yml

```diff
@@ -87,6 +98,13 @@ flows:
                     - when: "'npsp' not in tasks.get_installed_packages()"
                       action: hide
                 when: "org_config.has_minimum_package_version('npsp', '1.0')"
+            4:
+                task: site_template:install_managed_beta
```

Comment:
We can't install a beta in a customer org, so is this just for testing the step preflight?

Reply:
@jstvz yup! we can swap this out with `install_managed` before we merge this to `feature/231`

Reply:
@seanpat09 , do we have a Work Item to swap this out?   If not, can we add one?

Reply:
@spelak-salesforce would that be covered by this work item? I was under the impression that this item wouldn't get closed until we swap it out

Reply:
Yes as long as this does not merge until the `install_mangaged` task is referenced instead of the beta.   This will be easy to miss and merge without us realizing the installer doesn't work as expected.